stdlib/LinearAlgebra/src/adjtrans.jl

```diff
@@ -186,6 +186,16 @@ IndexStyle(::Type{<:AdjOrTransAbsMat}) = IndexCartesian()
 convert(::Type{Adjoint{T,S}}, A::Adjoint) where {T,S} = Adjoint{T,S}(convert(S, A.parent))
 convert(::Type{Transpose{T,S}}, A::Transpose) where {T,S} = Transpose{T,S}(convert(S, A.parent))
 
+# Strides and pointer for transposed strided arrays — but only if the elements are actually stored in memory
+Base.strides(A::Adjoint{<:Real, <:StridedVector}) = (stride(A.parent, 2), stride(A.parent, 1))
```

Comment:
This should not restrict to `<: StridedVector`. The strided array interface means other array types should be supported.

Reply:
Yes, I fully agree, but this is intended as a stopgap and not a full solution.